sbt/src/main/scala/stryker4s/sbt/runner/ProcessHandler.scala

```diff
@@ -0,0 +1,18 @@
+package stryker4s.sbt.runner
+
+import scala.tools.nsc.io.Socket
+import stryker4s.model.MutantRunResult
+import java.nio.file.Path
+
+class ProcessHandler() {
+  def newProcess(classpath: Seq[String]) = {
+    val socket = new Socket(new java.net.Socket("127.0.0.1", 13337))
```

Comment:
Old school. Inter process communication using sockets?

Reply:
Yeah... If it works don't fix it? The issue is I need some way to talk to talk to a process that is:

- Two-way communication
- Not stdin. Anything can write to stdin using `println`
- Doesn't introduce new dependencies. Could cause conflicts with the system under test
- Able to serialize jvm objects through. JSON/protobuf/anything else introduces new dependencies, see point 2

So I ended up with sockets. It's very low-level, but I'm open to suggestions haha